lib/openscap_parser/value.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+module OpenscapParser
+  class Value < XmlNode
+    include OpenscapParser::Util
+
+    def id
+      @id ||= parsed_xml['id']
+    end
+
+    def description
+      @description ||= newline_to_whitespace(parsed_xml.at_css('description')&.text)
+    end
+
+    def title
+      @title ||= parsed_xml.at_css('title')&.text
+    end
+
+    def type
+      @type ||= parsed_xml['type'] || 'string'
+    end
+
+    def generic_selector(type, selector = nil)
+      unless instance_variable_get("@#{type}")
+        element_name = type.to_s.gsub('_', '-')
```

Comment:
You don't need to run `gsub` that's a global `sub` and you're just swapping one char:
```suggestion
        element_name = type.to_s.sub('_', '-')
```